angular/projects/spark-angular/src/lib/components/sprk-accordion-item/sprk-accordion-item.component.ts

```diff
@@ -19,106 +21,145 @@ import { toggleAnimations } from '../sprk-toggle/sprk-toggle-animations';
       >
         <span [ngClass]="getHeadingClasses()">
           <sprk-icon
-            [iconType]="leadingIcon"
-            additionalClasses="sprk-c-Icon--filled-current-color sprk-c-Icon--xl sprk-u-mrs"
             *ngIf="leadingIcon"
+            [iconType]="leadingIcon"
+            additionalClasses="
+              sprk-c-Icon--filled-current-color
+              sprk-c-Icon--xl
+              sprk-u-mrs"
           ></sprk-icon>
-          {{ title }}
+          {{ heading || title }}
         </span>
 
         <sprk-icon
-          additionalClasses="
-            sprk-c-Accordion__icon
-            sprk-c-Icon--xl
-            sprk-c-Icon--toggle
-            {{iconStateClass}}"
+          [additionalClasses]="getIconClasses()"
           [iconType]="currentIconType"
```

Comment:
Update iconType to iconName.  Also, I know it's internal, but let's update `currentIconType` to be `currentIconName`